packages/code/src/Code.tsx

```diff
@@ -80,6 +94,86 @@ function getWrapperVariantStyle(variant: Variant): string {
   `;
 }
 
+function getSidebarVariantStyle(variant: Variant): string {
+  const colors = variantColors[variant] ?? variantColors[Variant.Light];
```

Comment:
Tip – you can destructure arrays similar to objects. Instead of this, you could do:

```js
const [, variantColor] = variantColors[variant] ?? variantColors[Variant.Light];

switch (variant) {
    case Variant.Light:
      return css`
        border-color: ${variantColor};
        background-color: white;
      `;

    case Variant.Dark:
      return css`
        border-color: ${variantColor};
        background-color: ${variantColor};
      `;
}
```

Reply:
I tried this but got back the error: `Type 'Base16Palette' must have a '[Symbol.iterator]()' method that returns an iterator.`